labs/behaviors/constraint-validation_test.ts

```diff
@@ -45,6 +47,52 @@ describe('mixinConstraintValidation()', () => {
     }
   }
 
+  /**
+   * A validator that set customError flag to true
+   */
+  class CustomErrorValidator extends Validator<SelectState> {
+    private control?: HTMLInputElement;
+
+    protected override computeValidity(state: SelectState) {
+      if (!this.control) {
+        this.control = document.createElement('input');
+      }
+      this.control.setCustomValidity('validator custom error');
```

Comment:
This needs `this.control.required = state.required` to support the multi-validation test

Reply:
Same comment as above: we only care about customError as this is not working properly with the current codebase.